### PR TITLE
Test build on Travis CI

### PR DESCRIPTION
**Pull request by lucafavatella**
In order to enable Travis CI builds on your repository, merging this PR is not enough. Please refer to http://docs.travis-ci.com/user/getting-started/

The next step (not included in this PR) would be to teach Travis CI to upload the built jar on GitHub after taking a tag. Ref http://docs.travis-ci.com/user/deployment/releases/

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ erlc +debug_info -o src/main/resources/erlyberly/beam/ src/main/resources/erlybe
 
 # install floaty field
 git clone https://github.com/andytill/floaty-field.git
+$(cd floaty-field && git checkout 0048c66)
```
Comment (andytill) [inline setup.sh]: Is a commit hash here because there are no tags?  If so, I can create a tag on floaty-field and jinterface and use that instead.

Comment (lucafavatella) [inline setup.sh]: Correct - hash is there because there are no recent tags. Tag would be better.

**Comment by andytill** [comment]
I have tagged the repositories from master, the version matches that in the pom version from master.

https://github.com/andytill/floaty-field/releases/tag/1.0.1
https://github.com/andytill/jinterface/releases/tag/1.6.0

**Comment by lucafavatella** [comment]
Force-pushed code change that uses tags (using the `-b` option of recent git).

**Comment by andytill** [comment]
Setup a Travis account, it looks like it is building ok already https://travis-ci.org/andytill/erlyberly

**Comment by andytill** [comment]
And the build is green, thanks!

**Pull request merged**